### PR TITLE
minor changes to be current

### PR DESCRIPTION
**Pull request by mellybelly**
Minor changes to be up to date with current recommendations relating to social distancing

### CODE REVIEW

File: _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ Taken together, making safer every-day choices on average --- not just this week
...
 -   Takeout \> eat on restaurant patio \> eat in restaurant
 -   Video streaming options \> movie theatre
--   Outside events \> inside events (small events also are better than large ones)
+-   No group events \> Outside events \> inside events (small events also are better than large ones)
 -   Delivery/curbside pickup \> going in person
 -   Remote observance \> religious service attendance
 -   Birthday cupcakes \> birthday cake (if candle blowing is involved)
--   Drink your coffee/beer/cocktail at a table instead \> drink your coffee/beer/cocktail at the bar
+-   Take your coffee/beer/cocktail to go \> drink your coffee/beer/cocktail at the bar
 -   Live performers are going to be hard hit economically. Consider supporting their work on Patreon, directly via PayPal, or attend only the smaller shows. Promote them on social media. Buy their work directly.
```
Comment (nicolevasilevsky) [inline _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md]: I think shows are cancelled everywhere, yeah?

File: _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ Taken together, making safer every-day choices on average --- not just this week
 -   Non-contact sports are better than contact ones. E.g., Tennis is better than (\>) rugby
 -   Takeout \> eat on restaurant patio \> eat in restaurant
 -   Video streaming options \> movie theatre
```
Comment (nicolevasilevsky) [inline _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md]: Are any movie theaters open at the moment? Maybe just delete this?

**Comment by nditada** [comment]
@mellybelly You have a few corrections by @nicolevasilevsky that I think are important before merging this. You should see a commit suggestion button, explained here: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

File: _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md (hunk 1 above)
Comment (jaysonvirissimo) [reply _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md]: There are still live shows in Mexico, and there could be other places where we have potential users as well. I'd rather we just delete the part about "attend only the smaller shows" as that seems like something we shouldn't be endorsing given how crowded even smaller shows can get.

Comment (nicolevasilevsky) [reply _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md]: I agree to delete

Comment (nditada) [reply _sections/act_and_prepare/en/02-do/03-lower-your-overall-risk.md]: I think only @mellybelly can accept that deletion on this PR, but I'm not sure she saw my note below. I'll merge this PR and then do the mods immediately after.

**Pull request merged**